src/OpenTracing/NullTracer/NullSpanContext.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections.Generic;
+
+namespace OpenTracing.NullTracer
+{
+    public class NullSpanContext : ISpanContext
+    {
+        internal static readonly NullSpanContext Instance = new NullSpanContext();
+
+        private readonly Dictionary<string, string> _baggage = new Dictionary<string, string>();
```

Comment:
There's `Enumerable.Empty<Type>()` for read-only lists (which returns a cached object), but there's no equivalent for dictionaries (maps) because they are mutable.